deposit/admin.py

```diff
@@ -28,7 +28,9 @@
 
 class DepositRecordAdmin(admin.ModelAdmin):
     list_display = ('identifier', 'paper', 'user')
+    list_filter = ['repository']
     raw_id_fields = ('paper', 'user', 'oairecord')
+    search_fields = ('paper__pk', 'paper__title')
```

Comment:
This adds a simple search field above the list of DepositRecords. Is this fine enough @wetneb ? Showing the DepositRecordIDs in a PaperView is non-trivial (but doable, if needed).

Reply:
That's great, thanks a lot!